memcached.c

```diff
@@ -166,7 +168,16 @@ static rel_time_t realtime(const time_t exptime) {
     }
 }
 
+static void stats_report_latency(stat_cmd_t cmd, hires_time_t latency) {
+    if (cmd < STAT_CMD_COUNT) {
+        STATS_LOCK();
+        percentile_sampler_record_sample(&stats.per_cmd_stats[cmd].handle_latency, latency);
+        STATS_UNLOCK();
+    }
+}
+
 static void stats_init(void) {
+    size_t i;
```

Comment:
Why size_t? It's counting indexes, so I would assume int or short would be more appropriate.